packages/gestalt/src/SelectList.js

```diff
@@ -12,12 +12,11 @@ import Icon from './Icon.js';
 import styles from './SelectList.css';
 import { type AbstractEventHandler } from './AbstractEventHandler.js';
 
-// eslint-disable-next-line flowtype/require-exact-type
-type Options = Array<{
+type Options = Array<{|
```

Comment:
This is exact, but we should explicitly declare it as exact so that the downstream projects with [`exact-by-default`](https://flow.org/en/docs/config/options/#toc-exact-by-default-boolean) and [`ambiguous-object-type`](https://flow.org/en/docs/linting/rule-reference/#toc-ambiguous-object-type) both enabled will not throw errors.